modules/ROOT/pages/proxy-deploy-raml-oas-proxy.adoc

```diff
@@ -0,0 +1,28 @@
+= Building a RAML/OAS API Proxy
+
+API Manager 2.x allows you to use a RAML/OAS type of API Proxy, for all your Mule 4 based proxies.
+
+== Configuring your RAML/OAS API Proxy
+
+. In API Manager, navigate to *Manage API* to create your API Instance. +
+You can choose to either xref:manage-exchange-api-task.adoc[Creating an API instance by importing from Exchange] or xref:manage-exchange-api-task.adoc[Creating an API instance by importing on API Manager].
+. In *Asset Type*, select *RAML/OAS*.
+. To configure your API Proxy for this API instance you need to select *Endpoint with Proxy* in the *Managing Type* fields.
+.. Select your *Proxy Deployment Target*  from the available deployment targets displayed.
+.. Select the *Check this box if you are managing this API in Mule 4 or above* check mark.
+.. Type in your *Implementation URI*.
++
+image::proxy-deploy-raml-oas-proxy-85634.png[]
+. Click *Advanced Options*.
+. In *Proxy Version*, select *v2.0 (Improved API specification parser)*.
```

Comment:
it is not only Improved API specification parser but also native OAS support. Adding @chiragnarangmulesoft to review wording

Reply:
I think what is missing is a note that v1.0 is deprecated in favor of v2.0 and comment the EOL. @chiragnarangmulesoft do you know what's the stance of the EOL here?

Reply:
yes we should clearly call out that they should use v2.0 for any OAS specs. there is no update on EOL yet

Reply:
Hi @fermujica, can we add a note stating:
"v2.0 is the recommended version for OAS/RAML specs. Native OAS is only supported with version 2.0, if v1.0 is used with OAS spec it is translated to RAML"

Reply:
Please check this note: 031b286